### ISSUE TITLE
Suggestion: Showing results per Criteria and by aggregation group

### ISSUE THREAD

**Issue by philipaxe** [ices-tools-dev/COMPEAT#7]
In addition to having results per indicator, it would be nice (helpful) even to have ER values per criteria. This would make reporting easier/more transparent and prevent us having to do the extra bit of arithmetic by hand (which can always introduce mistakes).

Related to that, COMP (and even HEAT) use the 3 - 4 categories, similar to the older Commission Decision, coveriing "Causative factors", "Direct effects", "Indirect effects", "Others(?)". Another "nice to have" would be to show the assessment results of these aggregations. This could be a little problematic, as there are (I thik) differences between HELCOM & OSPAR in which criteria are in which groups. I think macroalgal depth limits are one of those that are unclear.

**Comment by HjalteParner** [comment]
I'm not sure I understand fully the request. Currently the aggregation is done and shown per Criteria in the assessment tab.

Perhaps our terminology differs?

Criteria is for me the 3-4 groups e.g. Nutrients, Direct effects, Indirect effects and Others.

Category is for me the 10 groups feeding into the Criteria's.

If you are thinking about the categories then I understand and think that would be a nice addition. I just have to figure out how it can be displayed in an intuitive way - the structure is already in place behind the scene.

**Comment by philipaxe** [comment]
Yep. It looks like our terminology differs. The 3 - 4 groups (Nutrients, Direct effects, Indirect effects and Others) come from the COMP, Cloern et al (2001)* and the original Commission Decision** from 2010. These are the basis of the COMP assessment and we need them.
The present Commission Decision 2017*** requires us to report on 8 Criteria which are almost the same as indicators, but require some aggregation. They then contribute to the 3 - 4 (category) groups. Is this any clearer?

*Cloern, J. E. (2001) Our evolving conceptual model of the coastal eutrophication problem. Mar. Ecol. Pro.g Ser. 210: 223-253.
** 2010/477/EU: Commission Decision of 1 September 2010 on criteria and methodological standards on good environmental status of marine waters (notified under document C(2010) 5956) Text with EEA relevance, https://eur-lex.europa.eu/legal-content/EN/TXT/?uri=CELEX%3A32010D0477%2801%29 
*** Commission Decision (EU) 2017/848 of 17 May 2017 laying down criteria and methodological standards on good environmental status of marine waters and specifications and standardised methods for monitoring and assessment, and repealing Decision 2010/477/EU (Text with EEA relevance. )

**Comment by HjalteParner** [comment]
Yes, I will change my criteria and categories around so it follow the Commission Decision and COMP.
